iothub/service/src/Jobs/JobRequest.cs

```diff
@@ -26,39 +26,52 @@ public class JobRequest
         public JobType JobType { get; set; }
 
         /// <summary>
-        /// Required if jobType is cloudToDeviceMethod.
         /// The method type and parameters.
         /// </summary>
+        /// /// <remarks>
+        /// Required if jobType is cloudToDeviceMethod.
+        /// </remarks>
         [JsonProperty(PropertyName = "cloudToDeviceMethod")]
         public CloudToDeviceMethod CloudToDeviceMethod { get; set; }
 
         /// <summary>
-        /// Required if jobType is updateTwin.
         /// The Update Twin tags and desired properties.
         /// </summary>
+        /// <remarks>
+        /// Required if jobType is updateTwin.
```

Comment:
jobType is the serialized type.

There are a few ways to approach this. As this is referencing an actual property that isn't otherwise related with this one, I'd see a link

```suggestion
        /// Required if <see cref="JobType"/> is <see cref="JobType.UpdateTwin"/>.
```

Alternatively, you could just use plain English, so it'd be "Required if the job type is 'UpdateTwin'" (or even <c>UpdateTwin</c>).

Reply:
Same for CloudToDeviceMethod.

Reply:
And below.